tools/king/king.xml

```diff
@@ -37,41 +13,26 @@
     --projection $projection
     #end if
 
-    #if $unrelated
-    --unrelated
-    #end if
-
-    #if $build
-    --build
-    #end if
-
-    #if $cluster
-    --cluster
-    #end if
-
-    #if $rplot
-    --rplot
-    #end if
-
+    $unrelated $build $cluster $rplot
     > $kingoutput
     ]]></command>
     <inputs>
 	<param name="genotype" type="data" format="bed" help="Binary genotype File" />
 	<param name="family" type="data" format="txt" help="Family File" />
 	<param name="map" type="data" format="tabular" help="Map File" />
 	<!-- Beginning of the optional paramters -->
-	<param name="related" type="boolean" checked="false" optional="true" label="Relatedness" />
-	<param name="duplicate" type="boolean" checked="false" optional="true" label="Duplicates" />
-	<param name="kinship" type="boolean" checked="false" optional="true" label="Kinship" />
-	<param name="ibdseg" type="boolean" checked="false" optional="true" label="IBDSeg" />
-	<param name="ibs" type="boolean" checked="false" optional="true" label="IBS" />
-	<param name="homog" type="boolean" checked="false" optional="true" label="Homog" />
+	<param name="related" type="boolean" truevalue="--relatedness" falsevalue="" label="Relatedness" />
+	<param name="duplicate" type="boolean" truevalue="--duplicate" falsevalue="" label="Duplicates" />
+	<param name="kinship" type="boolean" truevalue="--kinship" falsevalue="" label="Kinship" />
+	<param name="ibdseg" type="boolean" truevalue="--ibdseg" falsevalue="" label="IBDSeg" />
+	<param name="ibs" type="boolean" truevalue="--ibs" falsevalue="" label="IBS" />
+	<param name="homog" type="boolean" truevalue="--homog" falsevalue="" label="Homog" />
```

Comment:
could you change these to `arg`  as mentioned before? That way you can remove the repetitive truevalue and falsevalue attributes.